docs/getting-started-guides/kubeadm.md

```diff
@@ -213,199 +278,298 @@ kube-system   canal-node-77d0h                  2/3       CrashLoopBackOff    3
+scp root@<master ip>:/etc/kubernetes/admin.conf .
+kubectl --kubeconfig ./admin.conf proxy
+```
 
 You can now access the API Server locally at `http://localhost:8001/api/v1`
 
 ### (Optional) Installing a sample application
 
-As an example, install a sample microservices application, a socks shop, to put your cluster through its paces. Note that this demo does only work on `amd64`.
-To learn more about the sample microservices app, see the [GitHub README](https://github.com/microservices-demo/microservices-demo).
+As an example, install a sample microservices application, a socks shop, to put
+your cluster through its paces. Note that this demo does only work on `amd64`.
+To learn more about the sample microservices app, see the [GitHub
+README](https://github.com/microservices-demo/microservices-demo).
 
-    # kubectl create namespace sock-shop
-    # kubectl apply -n sock-shop -f "https://github.com/microservices-demo/microservices-demo/blob/master/deploy/kubernetes/complete-demo.yaml?raw=true"
+``` bash
+kubectl create namespace sock-shop
+kubectl apply -n sock-shop -f "https://github.com/microservices-demo/microservices-demo/blob/master/deploy/kubernetes/complete-demo.yaml?raw=true"
+```
 
-You can then find out the port that the [NodePort feature of services](/docs/user-guide/services/) allocated for the front-end service by running:
+You can then find out the port that the [NodePort feature of
+services](/docs/user-guide/services/) allocated for the front-end service by
+running:
 
-    # kubectl describe svc front-end -n sock-shop
-    Name:                   front-end
-    Namespace:              sock-shop
-    Labels:                 name=front-end
-    Selector:               name=front-end
-    Type:                   NodePort
-    IP:                     100.66.88.176
-    Port:                   <unset> 80/TCP
-    NodePort:               <unset> 31869/TCP
-    Endpoints:              <none>
-    Session Affinity:       None
+``` bash
+kubectl describe svc front-end -n sock-shop
+```
 
-It takes several minutes to download and start all the containers, watch the output of `kubectl get pods -n sock-shop` to see when they're all up and running.
+Output:
 
-Then go to the IP address of your cluster's master node in your browser, and specify the given port.
-So for example, `http://<master_ip>:<port>`.
-In the example above, this was `30001`, but it is a different port for you.
+```
+Name:                   front-end
+Namespace:              sock-shop
+Labels:                 name=front-end
+Selector:               name=front-end
+Type:                   NodePort
+IP:                     100.66.88.176
+Port:                   <unset> 80/TCP
+NodePort:               <unset> 31869/TCP
+Endpoints:              <none>
+Session Affinity:       None
+```
 
-If there is a firewall, make sure it exposes this port to the internet before you try to access it.
+It takes several minutes to download and start all the containers, watch the
```

Comment:
Replace comma with period. Start the resulting second sentence with:

"You can watch the output of..."